edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+## WCA Disciplinary Crash Course
+
+# Disciplinary
+
+The WCA Disciplinary Committee (WDC) is entitled to apply bans and other disciplinary action as a result of violation of the [WCA Code of Conduct](https://www.worldcubeassociation.org/documents/Code%20of%20Conduct.pdf), based on the motion [Suspensions and other Sanctions](https://www.worldcubeassociation.org/documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf). However, the Committee is not able to handle difficult situations quickly during a competition, especially if communication with multiple involved parties is required. Cooperation with competition Delegates is often necessary. This document [or chapter, or section, or whatever this is going to be] contains a collection of guidelines to instruct new Delegates on how to handle common disciplinary incidents, what disciplinary measures they can take and what they should report to the WDC.
+
+# Disciplinary Log and Banned Competitors
+
+* A list of banned competitors can be found in the [Disciplinary log}(https://www.worldcubeassociation.org/disciplinary). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WDC.
```

Comment:
The `}` should be a `]`.

Also, how should a delegate contact WDC? If they are a delegate, they don't (currently) have access to Slack. The Unofficial WCA Staff Discord is currently unofficial, and until it gets some other level of approval from Board+, it's probably best to not officially endorse it.

Reply:
We are leaving it intentionally open since situations where people are banned from attending can be more serious. Anytime communication is mentioned in the WCA it almost always refers to email so the delegates should already be aware of that option, but if they have a faster form of communication like the discord server, the contact info of a WDC member, etc. then they are welcome to use those methods as well

Reply:
Good point. With the Discord and Slack I kinda forgot email existed haha.